linux_os/guide/system/permissions/partitions/mount_option_boot_nosuid/rule.yml

```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Add nosuid Option to /boot'
+
+description: |-
+    The <tt>nosuid</tt> mount option can be used to prevent
+    execution of setuid programs in <tt>/boot</tt>. The SUID and SGID permissions
+    should not be required in these user data directories.
+    {{{ describe_mount(option="nosuid", part="/boot") }}}
+
+rationale: |-
+    The presence of SUID and SGID executables should be tightly controlled. Users
+    should not be able to execute SUID or SGID binaries from boot partitions.
+
+severity: unknown
```

Comment:
A severity should be set.

Reply:
@yuumasato, @stevegrubb, what should be the severity for these `mount_option_*` rules? We currently have `unknown` in most of them, with just `mount_option_dev_shm_*` having `medium`.

Reply:
Honestly, I don't know how the `severity` is set.